resident-ui-update-demographics-schema.json

```diff
@@ -55,116 +55,6 @@
       "tabgroup": "identity",
       "dataType": "notString"
     },
-    {
```

Comment:
what will happen if we change the schema that is having firstName.

Reply:
FormatOption property is used to show different formats of name dropdown if firstName is there it will show one more option in the dropdown.

Reply:
okay